build.gradle

```diff
@@ -34,35 +34,33 @@ allprojects {
     }
 
     repositories {
-        ['TBD'].each { repo ->
-            maven {
-                credentials {
-                    username = findProperty('user') ?: System.getenv('user')
-                    password = findProperty('password') ?: System.getenv('password')
-                }
-
-                url "TBD/$repo/"
+        maven {
+            credentials {
+                username = findProperty('ACCESS_TOKEN_USER') ?: System.getenv('ACCESS_TOKEN_USER')
+                password = findProperty('ACCESS_TOKEN_VALUE') ?: System.getenv('ACCESS_TOKEN_VALUE')
             }
+
+            url "https://maven.pkg.github.com/Alvearie/hri-flink-pipeline-core"
         }
         mavenCentral()
         mavenLocal()
     }
 
     ext {
-        branch = System.getenv('TRAVIS_BRANCH') != null
-                ? System.getenv('TRAVIS_BRANCH')
+        branch = System.getenv('BRANCH_NAME') != null
+                ? System.getenv('BRANCH_NAME')
                 : getWorkingBranch()
     }
 
-    // If not running in travis add 'local' to the version to support local development
-    if (System.getenv('TRAVIS_BRANCH') == null || System.getenv('TRAVIS_BRANCH') == "") {
+    // If not running in Actions add 'local' to the version to support local development
+    if (System.getenv('BRANCH_NAME') == null || System.getenv('BRANCH_NAME') == "") {
         version = "${branch}-local-SNAPSHOT"
-    } else if (System.getenv('TRAVIS_TAG') == null || System.getenv('TRAVIS_TAG') == "") {
+    } else if (System.getenv('ACTIONS_TAG') == null || System.getenv('ACTIONS_TAG') == "") {
         version = "${branch}-SNAPSHOT"
-    } else if (System.getenv('TRAVIS_TAG') == "v${version}") {
+    } else if (System.getenv('ACTIONS_TAG') == "v${version}") {
```

Comment:
I think you are missing setting teh ACTIONS_TAG anywhere in your existing workflows?  Should that be in the ci-workflow.yml or in another workflow (r`elease.yml)`. 

That is are you missing the `release.yml` workflow, which publishes the artifacts on tags

Reply:
Whoops, you're right I missed the ACTIONS_TAG. We decided to not have a separate release.yml workflow for this one because there was a large amount of overlap. 

But obviously needed to set the variable then heh, should be fixed now!

Reply:
Yup. Looks better now.